h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -94,6 +101,34 @@ public void addScoringInfo(GLMParameters parms, int nclasses, long currTime, int
   
   public void setVcov(double[][] inv) {_output._vcov = inv;}
 
+  /***
+   * This method will calculate the variable inflation factor of each numerical predictor using the following
```

Comment:
Do we really need to implement this procedure as part of GLM model training? 

It can be calculated after the model was already built - and we would not need to add additional parameters to GLM, wdyt?

Reply:
The advantage is in CV - we will have the values also for CV models and user doesn't need to figure out what was the fold assignment.